classes/class-wc-connect-shipping-method.php

```diff
@@ -290,7 +290,8 @@ public function calculate_shipping( $package = array() ) {
 							)
 						),
 						'cost'     => $rate->rate,
-						'calc_tax' => 'per_item'
+						'calc_tax' => 'per_item',
+						'meta_data' => array( 'wc_connect_packages' => json_encode( $rate->packages ) ),
```

Comment:
According to the method signature of `WC_Shipping_Rate->add_meta_data()` , both the key and the value must be `string`. I don't know where it's being all serialized, so I just wanted to play it safe here :)